test/jdk/com/sun/jdi/DeferredStepTest.java

```diff
@@ -74,8 +74,17 @@ public static void  main(String argv[]) {
 
         jj1 obj1 = new jj1();
         jj2 obj2 = new jj2();
-        TestScaffold.newThread(obj1, "jj1").start();
-        TestScaffold.newThread(obj2, "jj2").start();
+        Thread jj1 = TestScaffold.newThread(obj1, "jj1");
+        Thread jj2 = TestScaffold.newThread(obj2, "jj2");
+        jj1.start();
+        jj2.start();
```

Comment:
It looks strange that there are errors about conflict between variable names and class names.
Anyway I think it would be better to rename the variables